rust/lance-encoding/src/encodings/physical/bitpack_fastlanes.rs

```diff
@@ -1560,3 +1564,251 @@ mod tests {
     //     check_round_trip_bitpacked(arr).await;
     // }
 }
+
+// This macro chunks the FixedWidth DataBlock, bitpacks them with 1024 values per chunk,
+// it puts the bit-width parameter in front of each chunk,
+// and the bit-width parameter has the same bit-width as the uncompressed DataBlock
+// for example, if the input DataBlock has `bits_per_value` of `16`, there will be 2 bytes(16 bits)
+// in front of each chunk storing the `bit-width` parameter.
```

Comment:
Why not just always use 1 byte?

Reply:
yeah, this is related to the type of `output` vector, it's type is the same as input data type(to satisfy signature of Bitpack::unchecked_pack), so when we are filling in `bit-width` into `output`, it's type is same as `input data type`

Reply:
I think for fastlanes bitpack compression to work, its start output buffer position must be aligned to the same input data type alignment.

Reply:
I think it will panic to squeeze in a `u8` into a vector of `u64` and then treat the rest of vector as `u64`, because the alignment requirement doesn't hold anymore.